webui/react/src/types.ts

```diff
@@ -197,6 +197,16 @@ interface ExperimentHyperParam {
 
 export type ExperimentHyperParams = Record<string, ExperimentHyperParam>;
 
+export enum ExperimentSearcherName {
+  AdaptiveAdvanced = 'adaptive',
```

Comment:
@katport does this look good to you? are these all the experiment searcher names? it looks to be all of what https://docs.determined.ai/latest/reference/experiment-config.html#experiment-configuration-searcher talks about

Reply:
I got these directly from the docs, but I couldn't actually find any examples in our repot for `pbt`, `random` or `grid`...